Modelica/Magnetic/QuasiStatic/FluxTubes/package.mo

```diff
@@ -11,8 +11,7 @@ package FluxTubes "Library for modelling of quasi-static electromagnetic devices
 This library is intended to provide models for the investigation of
 quasi-static electromagnetic devices with lumped magnetic networks.
 </p>
-</html>"),
-           Icon(coordinateSystem(preserveAspectRatio=false), graphics={
+</html>"), Icon(coordinateSystem(preserveAspectRatio=false), graphics={
```

Comment:
```suggestion
</html>"),
    Icon(coordinateSystem(preserveAspectRatio=false), graphics={
```

Reply:
I think it is good, but would prefer a general cleanup of those.

Reply:
Ah well, adding it anyway.